### DISCUSSION TITLE
Meeting Agenda 2/8

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#273] [Meeting Agendas/Minutes]
1. Review tasks
2. Show diagrams to Darian

**Comment by moreno-michael** [comment]
Meeting minutes 2/8

1. Task review
- Issue #1262 still in progress
- Issue #1239 complete, #1295 still in progress
- Issues #1296, #1297 still in progress
2. Spoke to Darian
- Keep track of what we are working on outside of sprint planning to keep him in the loop and update him with every meeting (show him our meeting minutes)
- Chris: Make a diagram explaining the plugin interaction with the dom
- Michael: update DNSMonolithDiscover class diagram to account for file updates, make an activity diagram for issue #1295
- Victor: Update figure 31 to show that all classes are contained within monolith, clarify the activity diagram
- Add diagrams to spec doc
